zipkin.proto

```diff
@@ -215,6 +213,10 @@ message ListOfSpans {
   repeated Span spans = 1;
 }
 
+// Response for SpanService/Report RPC. This response currently does not return
+// any information beyond indicating that the request has finished.
```

Comment:
Can consider adding `, but may be extended in the future` if the flexibility might help later

Reply:
sounds fine to me